src/Sulu/Bundle/TestBundle/Resources/dist/parameters.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+    database_mysql_host:     localhost
+    database_mysql_user:     root
+    database_mysql_password: ~
+    database_mysql_port:     ~
+
+    database_pgsql_host:     localhost
+    database_pgsql_user:     postgres
+    database_pgsql_password: ~
+    database_pgsql_port:     ~
+
+    phpcr_backend_url: http://localhost:8080/server/
+    phpcr_username:    admin
+    phpcr_password:    admin
```

Comment:
I'd quite like two. I'm testing against both Postgres and MySQL — in separate runs I grant — but with different settings I just need to adjust a single environment variable and I'm done.